lib/bullet_train/outgoing_webhooks/engine.rb

```diff
@@ -27,7 +27,7 @@ class Engine < ::Rails::Engine
       end
 
       initializer "bullet_train.outgoing_webhooks.register_api_endpoints" do |app|
-        if BulletTrain::Api
+        if Object.const_defined?('BulletTrain::Api')
```

Comment:
https://stackoverflow.com/a/31258221

Required for `rails test` to execute and should probably be copypasta across all of the BulletTrain gems that use this pattern.